SPECS/python-filelock/python-filelock.spec

```diff
@@ -0,0 +1,146 @@
+%global srcname filelock
+
+Summary:        A platform independent file lock
+Name:           python-%{srcname}
+Version:        3.0.12
+Release:        10%{?dist}
+License:        Unlicense
+URL:            https://github.com/benediktschmitt/py-filelock
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+#Source0:       https://github.com/benediktschmitt/py-%{srcname}/archive/v%{version}/py-%{srcname}-%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
```

Comment:
A neat "trick" I've learned from a different spec yesterday:

```
Source0:       [https://github.com/benediktschmitt/py-%{srcname}/archive/v%{version}/py-%{srcname}-%{version}.tar.gz#/%{name}-%{version}.tar.gz](https://github.com/benediktschmitt/py-%{srcname}/archive/v%{version}/py-%{srcname}-%{version}.tar.gz#/%{name}-%{version}.tar.gz)
```

This way `rpmbuild` still tries to extract `%{name}-%{version}.tar.gz` but the URL is now easily retrievable after running `rpmspec`. No need to change anything, just wanted to let you know, in case you like it too.